tests/multi_ptr/multi_ptr_accessor_constructor_fp16.cpp

```diff
@@ -33,10 +32,10 @@ class TEST_NAME : public sycl_cts::util::test_base {
 
     auto queue = util::get_cts_object::queue();
 
-    using avaliability =
-        util::extensions::availability<util::extensions::tag::fp16>;
-    if (!avaliability::check(queue, log)) {
-      SKIP("Device does not support half precision floating point operations");
+    if (!queue.get_device().has(sycl::aspect::fp16)) {
+      SKIP(
+          "Device does not support half precision floating point operations - "
+          "skipping the test");
```

Comment:
```suggestion
          "Device does not support half precision floating point operations");
```
I don't think we need explicitly add "- skipping the test" for SKIP macro.

We should avoid using WARN + return idiom in other tests and use SKIP instead.

Reply:
To clarify my motivation here - some tests are still using legacy test framework and I wasn't sure how `SKIP` (that comes from `Catch2`) interacts with it. Therefore, to be conservative I've avoided using `SKIP` in those tests.